README.md

```diff
@@ -90,6 +90,10 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
  
   [<img src = "/art/particle.gif" width=240>](http://dartpad.dev/6613d586f958bf692f17b4407fe4e2cb)
 
+- [Todo app](https://gist.github.com/fayaz07/16c9149a3775c6c289625b89d10c62ef) by [Mohammad Fayaz](https://github.com/fayaz07) 
```

Comment:
Please add a date at the end.

Reply:
Thank you for reviewing my PR, I have also written Unit, Widget and Integration tests to this, shall I add a link at the end?